acceptancetests/clean_prodstack.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import re
+import os
+import sys
+import logging
+import subprocess
+
+def get_current_controller_and_models():
+    lines = subprocess.check_output(
+        ['juju', 'models'], stderr=subprocess.STDOUT).split('\n')
+    ctrl_name = [
+        item.split(':')[-1].strip() for item in lines if 'Controller:' in item][0]
+    for i in range(0, len(lines)):
+        if ('Model' in lines[i] and
+            'Cloud/Region' in lines[i] and
+            i+1<len(lines)):
+            model_lines = filter(None, lines[i+1:])
+            break
+    if not model_lines:
+        logging.info('No model has been found, exit.')
+        sys.exit(1)
+    model_list = [elem.split(' ')[0].strip('*') for elem in model_lines]
+    logging.info(
+        'Models under the Controller {} are:\n{}'.format(
+        ctrl_name,
+        '\n'.join(model_list)))
+    return (ctrl_name, model_list)
+
+
+def get_reserved_machine_ipaddress():
+    role = 'admin'
+    ip_pool = []
+    ctrl_name, model_list = get_current_controller_and_models()
+    for item in model_list:
```

Comment:
model is a better variable name then item, model_name is even better as it describes what it is.